src/index.ts

```diff
@@ -1,12 +1,12 @@
 import { Worker, MessageChannel, MessagePort, receiveMessageOnPort } from 'worker_threads';
 import { once } from 'events';
-import EventEmitterAsyncResource from 'eventemitter-asyncresource';
+import EventEmitterAsyncResource = require('eventemitter-asyncresource');
 import { AsyncResource } from 'async_hooks';
 import { cpus } from 'os';
 import { fileURLToPath, URL } from 'url';
 import { resolve } from 'path';
 import { inspect, types } from 'util';
-import assert from 'assert';
+import assert = require('assert');
```

Comment:
`assert` also gives you the same error?

Reply:
yeah. can do `import * as assert from 'assert'`, but `assert` is used as a function, so that breaks. `import { strict as assert } from 'assert'` possibly?

Reply:
or rather, this doesn't give an error in _my_ code, as the import is removed from the `d.ts` file. It just gives an error if turning off `esModuleInterop`. So the change can be removed from this PR and still solve my issue

Reply:
Let's revert this one to scope only `eventemitter-asyncresource` for the PR. We can visit the `assert` in #375, wdyt?